config/sidekiq_scheduler.yml

```diff
@@ -196,7 +196,7 @@ AppsApi::FetchConnections:
   description: "Fetches and handles notifications for recent application connections and disconnections"
 
 EducationForm::Process10203Submissions:
-  cron: "0 6-18 * * * America/New_York"
+  cron: "0 6-18/6 * * * America/New_York"
```

Comment:
This will run at 6am, 12 noon, and 6pm every day including weekends.

Previously this ran 12 times a day, now it will run three.

Reply:
Yeah, was wondering if we want to get rid of the `6-18` and let it run 4 times?